sklearn/linear_model/tests/test_ridge.py

```diff
@@ -527,6 +529,29 @@ def test_ridgecv_store_cv_values():
     assert_equal(r.cv_values_.shape, (n_samples, n_responses, n_alphas))
 
 
+def test_ridgecv_sample_weight():
+    rng = np.random.RandomState(0)
+    alphas = (0.1, 1.0, 10.0)
+
+    for n_samples, n_features in ((6, 5), (5, 10)):
```

Comment:
> Hmm, what does this mean, "2 options are faster in each case"? Faster relative
> to what? Sorry, really new to the code base, may ask stupid questions :)

There are different algorithms for the n_samples > n_features case and
the other one, that are optimal in the two different cases.